tests/pytests/test_vecsim.py

```diff
@@ -98,34 +98,32 @@ def testDel(env):
+        env.assertEqual(res[1:5], ['a', ['v_score', '0', 'v', 'aaaaaaaa'], 'b', ['v_score', '3.09485009821e+26', 'v', 'aaaabaaa']])
         
-        res = env.cmd('FT.SEARCH', 'idx', '@v:[abcdefgh TOPK 3]', 'SORTBY', 'v_score', 'ASC', 'LIMIT', 0, 3)
-        env.assertEqual(res[1:7], ['a', ['v_score', '0', 'v', 'abcdefgh'], 'c', ['v_score', '8.30767497366e+34', 'v', 'aacdefgh'],
-                                   'd', ['v_score', '4.78522078483e+37', 'v', 'azcdefgh']])
+        res = env.cmd('FT.SEARCH', 'idx', '@v:[aaaaaaaa TOPK 3]', 'SORTBY', 'v_score', 'ASC', 'LIMIT', 0, 3)
+        env.assertEqual(res[1:7], ['a', ['v_score', '0', 'v', 'aaaaaaaa'], 'b', ['v_score', '3.09485009821e+26', 'v', 'aaaabaaa'],
+                                   'c', ['v_score', '2.02824096037e+31', 'v', 'aaaaabaa']])
         
-        res = env.cmd('FT.SEARCH', 'idx', '@v:[abcdefgh TOPK 4]', 'SORTBY', 'v_score', 'ASC', 'LIMIT', 0, 4)
-        env.assertEqual(res[1:9], ['a', ['v_score', '0', 'v', 'abcdefgh'], 'c', ['v_score', '8.30767497366e+34', 'v', 'aacdefgh'],
-                                   'd', ['v_score', '4.78522078483e+37', 'v', 'azcdefgh'], 'b', ['v_score', 'inf', 'v', 'abcdefgg']])
+        res = env.cmd('FT.SEARCH', 'idx', '@v:[aaaaaaaa TOPK 4]', 'SORTBY', 'v_score', 'ASC', 'LIMIT', 0, 4)
+        env.assertEqual(res[1:9], ['a', ['v_score', '0', 'v', 'aaaaaaaa'], 'b', ['v_score', '3.09485009821e+26', 'v', 'aaaabaaa'],
+                                   'c', ['v_score', '2.02824096037e+31', 'v', 'aaaaabaa'], 'd', ['v_score', '1.32922799578e+36', 'v', 'aaaaaaba']])
         
         conn.execute_command('DEL', 'a')
         
-        res = env.cmd('FT.SEARCH', 'idx', '@v:[abcdefgh TOPK 1]', 'SORTBY', 'v_score', 'ASC', 'LIMIT', 0, 1)
-        env.assertEqual(res[1:3], ['c', ['v_score', '8.30767497366e+34', 'v', 'aacdefgh']])
-        res = env.cmd('FT.SEARCH', 'idx', '@v:[abcdefgh TOPK 2]', 'SORTBY', 'v_score', 'ASC', 'LIMIT', 0, 2)
-        env.assertEqual(res[1:5], ['c', ['v_score', '8.30767497366e+34', 'v', 'aacdefgh'], 'd', ['v_score', '4.78522078483e+37', 'v', 'azcdefgh']])
-        res = env.cmd('FT.SEARCH', 'idx', '@v:[abcdefgh TOPK 3]', 'SORTBY', 'v_score', 'ASC', 'LIMIT', 0, 3)
-        env.assertEqual(res[1:7], ['c', ['v_score', '8.30767497366e+34', 'v', 'aacdefgh'], 'd', ['v_score', '4.78522078483e+37', 'v', 'azcdefgh'],
-                        'b', ['v_score', 'inf', 'v', 'abcdefgg']])
+        res = env.cmd('FT.SEARCH', 'idx', '@v:[aaaaaaaa TOPK 1]', 'SORTBY', 'v_score', 'ASC', 'LIMIT', 0, 1)
+        env.assertEqual(res[1:3], ['b', ['v_score', '3.09485009821e+26', 'v', 'aaaabaaa']])
+        res = env.cmd('FT.SEARCH', 'idx', '@v:[aaaaaaaa TOPK 2]', 'SORTBY', 'v_score', 'ASC', 'LIMIT', 0, 2)
+        env.assertEqual(res[1:5], ['b', ['v_score', '3.09485009821e+26', 'v', 'aaaabaaa'], 'c', ['v_score', '2.02824096037e+31', 'v', 'aaaaabaa']])
```

Comment:
If I'm not mistaking, in all of the tests, the results are ordered by their id. Consider adding a test where the expected order of the results when you `SORTBY v_score` is _not_ the ascending oreder of the ids as well (maybe it also related to the bug that Ed found..)